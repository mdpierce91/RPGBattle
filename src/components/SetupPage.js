import React from 'react';
import TeamBuilder from './TeamBuilder';

const SetupPage = () => {
    return (
        <div>
            <TeamBuilder />
            <button>Find Battle!</button>
        </div>
    );
}

export { SetupPage as default };