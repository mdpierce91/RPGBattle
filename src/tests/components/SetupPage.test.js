import React from 'react';
import { shallow } from 'enzyme';
import SetupPage from '../../components/SetupPage';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<SetupPage/>);
});

test('should render setup page', () => {
    expect(wrapper).toMatchSnapshot();
})