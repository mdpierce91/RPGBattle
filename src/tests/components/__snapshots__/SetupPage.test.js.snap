// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render setup page 1`] = `
<div>
  <TeamBuilder />
  <button>
    Find Battle!
  </button>
</div>
`;
